content/pages/kevin/index.mdx

```diff
@@ -4,8 +4,10 @@ slug: "/kevin"
 ---
 import ProfileHeader from "../../../src/components/profile-header"
 import Button from "../../../src/components/button"
+import AuthorProjects from "../../../src/components/author-projects"
+import KevinProfileImage from "./kevin.jpeg"
 
-<ProfileHeader name="kevin.jpeg" title="Kevin">
+<ProfileHeader image={KevinProfileImage} title="Kevin">
```

Comment:
Just adding the image object thing I was talking about rather than the name of the image.